### PR TITLE
CloudFormation: Add support for template configuration

### PR DESCRIPTION
**Pull request by crkoiral**
*Issue #, if available:*
N/A

*Description of changes:*
Add support for template configuration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ishaan-berry — APPROVED** [review]

File: aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/Translator.java [hunk 1]
```diff
@@ -13,16 +13,7 @@
...
-import software.amazon.kendra.datasource.convert.GoogleDriveConverter;
-import software.amazon.kendra.datasource.convert.ListConverter;
-import software.amazon.kendra.datasource.convert.OneDriveConverter;
-import software.amazon.kendra.datasource.convert.S3Converter;
-import software.amazon.kendra.datasource.convert.ServiceNowConverter;
-import software.amazon.kendra.datasource.convert.SharePointConverter;
-import software.amazon.kendra.datasource.convert.SalesforceConverter;
-import software.amazon.kendra.datasource.convert.WebCrawlerConverter;
-import software.amazon.kendra.datasource.convert.WorkDocsConverter;
+import software.amazon.kendra.datasource.convert.*;
```
Comment (tzamazon) [inline aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/Translator.java:16]: maybe we should avoid import with *?

File: aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/utils/DocumentTypeAdapter.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package software.amazon.kendra.datasource.utils;
+
+import com.google.gson.*;
```
Comment (tzamazon) [inline aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/utils/DocumentTypeAdapter.java:3]: similar to the comment above.

File: aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/Translator.java (hunk 1 above)
Comment (crkoiral) [reply aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/Translator.java]: Please review, I have fixed it.

File: aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/utils/DocumentTypeAdapter.java (hunk 2 above)
Comment (crkoiral) [reply aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/utils/DocumentTypeAdapter.java]: Please review, I have fixed it.

**Review by tzamazon — APPROVED** [review]

**Review by ishaan-berry — APPROVED** [review]

**Pull request merged**